[2022-10-15 14:38:29,905] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: gpu_test.check_gpu manual__2022-10-15T13:18:12.870163+00:00 [queued]>
[2022-10-15 14:38:29,914] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: gpu_test.check_gpu manual__2022-10-15T13:18:12.870163+00:00 [queued]>
[2022-10-15 14:38:29,915] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-15 14:38:29,916] {taskinstance.py:1357} INFO - Starting attempt 6 of 6
[2022-10-15 14:38:29,917] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-15 14:38:29,927] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): check_gpu> on 2022-10-15 13:18:12.870163+00:00
[2022-10-15 14:38:29,932] {standard_task_runner.py:52} INFO - Started process 1523 to run task
[2022-10-15 14:38:29,934] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'gpu_test', 'check_gpu', 'manual__2022-10-15T13:18:12.870163+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/gpu_test.py', '--cfg-path', '/tmp/tmpiwp0zxjt', '--error-file', '/tmp/tmpmfetq64q']
[2022-10-15 14:38:29,935] {standard_task_runner.py:80} INFO - Job 15: Subtask check_gpu
[2022-10-15 14:38:29,992] {task_command.py:369} INFO - Running <TaskInstance: gpu_test.check_gpu manual__2022-10-15T13:18:12.870163+00:00 [running]> on host 44f12f161b19
[2022-10-15 14:38:30,048] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gpu_test
AIRFLOW_CTX_TASK_ID=check_gpu
AIRFLOW_CTX_EXECUTION_DATE=2022-10-15T13:18:12.870163+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-15T13:18:12.870163+00:00
[2022-10-15 14:38:30,050] {python.py:173} INFO - Done. Returned value was: <module 'docker' from '/home/airflow/.local/lib/python3.7/site-packages/docker/__init__.py'>
[2022-10-15 14:38:30,056] {xcom.py:585} ERROR - Could not serialize the XCom value into JSON. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config.
[2022-10-15 14:38:30,058] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 68, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2380, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 68, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 197, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 582, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/local/lib/python3.7/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type module is not JSON serializable
[2022-10-15 14:38:30,067] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=gpu_test, task_id=check_gpu, execution_date=20221015T131812, start_date=20221015T143829, end_date=20221015T143830
[2022-10-15 14:38:30,074] {standard_task_runner.py:97} ERROR - Failed to execute job 15 for task check_gpu (Object of type module is not JSON serializable; 1523)
[2022-10-15 14:38:30,110] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-10-15 14:38:30,140] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
